packages/loader/container-loader/src/retriableDocumentStorageService.ts

```diff
@@ -87,8 +109,15 @@ export class RetriableDocumentStorageService implements IDocumentStorageService
                 }
                 // If it is not retriable, then just throw the error.
                 if (!canRetryOnError(err)) {
+                    this.logger.sendErrorEvent({
+                        eventName: `StorageCallsNonRetriable_${fetchCallName}`,
+                        numRetries,
+                        totalTime: TelemetryLogger.formatTick(performance.now() - startTime),
```

Comment:
I think you should and can use duration property here, and it will be auto-formatted for you.

Reply:
Please double check that though